packages/now-cli/src/util/domains/setup-domain.ts

```diff
@@ -18,22 +18,14 @@ export default async function setupDomain(
   contextName: string
 ) {
   const aliasDomain = extractDomain(alias);
-  const parsedDomain = psl.parse(aliasDomain);
-  if (parsedDomain.error) {
-    return new ERRORS.InvalidDomain(alias, parsedDomain.error.message);
-  }
-  if (!parsedDomain.domain) {
-    return new ERRORS.InvalidDomain(alias);
-  }
-
-  const { domain } = parsedDomain;
-  output.debug(`Trying to fetch domain ${domain} by name`);
-  const info = await maybeGetDomainByName(client, contextName, domain);
+  output.debug(`Trying to fetch domain ${aliasDomain} by name`);
+  const info = await maybeGetDomainByName(client, contextName, aliasDomain);
   if (info instanceof ERRORS.DomainPermissionDenied) {
     return info;
   }
 
   if (info) {
+    const { name: domain } = info;
```

Comment:
Whats the difference between `aliasDomain` and `info.name`?

Reply:
`aliasDomain` can be a subdomain of `info.name` and `info.name` will be the apex domain the user controls @styfle